### PR TITLE
feat(skyrim5-client): set in-game ActorValues to the values specified in `createActor` packet on the client

### LINKED ISSUE #187: Set in-game ActorValues to the values specified in `createActor` packet on the client
Tip: Use `GetAV` console command in game to check if values are applied correctly.

### PR DESCRIPTION
**Pull request by ZikkeyLS**
Closes #187

### CODE REVIEW

**Comment by Pospelove** [comment]
@SupAidmi  @ZikkeyLS We do not review drafts since drafts can not be merged. The author needs to mark the pull request as ready for review so we can go forward.

File: skymp5-client/src/front/components/actorvalues.ts [hunk 1]
```diff
@@ -18,3 +18,13 @@ export const getActorValues = (ac: Actor): ActorValues => {
...
+
+let baseActorValues: Map<string, unknown>; 
+
+export const setBaseActorValues = (baseAV: Map<string, unknown>): void => {
+    baseActorValues = baseAV;
+}
+
+export const getBaseActorValues = (): Map<string, unknown> => {
+    return baseActorValues;
+}
```
Comment (alekcey0211) [inline skymp5-client/src/front/components/actorvalues.ts:30]: And what’s the point of these two functions?

**Review by alekcey0211 — COMMENTED** [review]
Check my comment. I don’t understand why these functions.

File: skymp5-client/src/front/components/actorvalues.ts (hunk 1 above)
Comment (ZikkeyLS) [reply skymp5-client/src/front/components/actorvalues.ts]: discussed personally with @alekcey0211

**Review by alekcey0211 — APPROVED** [review]
I think everything is ok

**Review by q1000treadz — APPROVED** [review]

**Pull request merged**